plugins/automation/spec/scripts/add_user_to_group_through_custom_field_spec.rb

```diff
@@ -117,4 +133,40 @@
       end
     end
   end
+
+  context "with many users and groups" do
+    fab!(:bangalore) { Fabricate(:group, full_name: "Bangalore, India") }
+    fab!(:dublin) { Fabricate(:group, full_name: "Dublin, Ireland") }
+    fab!(:iowa) { Fabricate(:group, full_name: "Iowa") }
+    fab!(:missouri) { Fabricate(:group, full_name: "Missouri") }
+
+    fab!(:user1) { Fabricate(:user, id: bangalore.id) }
```

Comment:
@nattsw what's the purpose of specifying id here? (it's causing flakes, since there sometimes exists a user with that id)